src/assets/styles/PetForm.css

```diff
@@ -0,0 +1,214 @@
+.petFormContainer {

```

Comment:
This is practically the same as petFormContainer2 so you can use one petFormContainer for both implementation and create new selector with variations (grid-template-columns: 1fr 1fr;)

Reply:
Iv'e just implemented a selector only for grid-template-columns: 1fr 1fr, and used only one petContainer